### PR TITLE
Add FoodMagic ✨

### PR DESCRIPTION
**Pull request by Sameerkash**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,8 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 
 * **Godot + Appwrite** - A demo app to showcase Godot's integration with Appwrite. [Source Code](https://github.com/lohanidamodar/godot-appwrite)
 
+* **FoodMagic ✨** - An Augmented Reailty Food Delivery app built using Flutter and Appwrite ❤️. [Source Code](https://github.com/Sameerkash/FoodMagic)
```
Comment (christyjacob4) [inline README.md:163]: I'd recommend you to add this under the 30DaysOfAppwrite section along with the other projects.

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please remove the extra line here :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -160,6 +162,7 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 
 * **Godot + Appwrite** - A demo app to showcase Godot's integration with Appwrite. [Source Code](https://github.com/lohanidamodar/godot-appwrite)
 
+
```
Comment (christyjacob4) [inline README.md:165]:
```suggestion
```

**Comment by Sameerkash** [comment]
@christyjacob4 could you take a look?

**Review by christyjacob4 — APPROVED** [review]

**Comment by eldadfux** [comment]
Great work @Sameerkash ❤️

**Comment by Sameerkash** [comment]
Thanks @eldadfux 
I'll be contributing more soon <3

**Pull request merged**